plugins/plugin-eval/src/plugin_eval/layers/static.py

```
@@ -27,6 +27,23 @@ def anti_pattern_penalty(count: int) -> float:
+    Skills that opt out of model-driven invocation (`disable-model-invocation:
+    true`) or that auto-load on a path glob (`paths:` frontmatter) use a
+    different trigger mechanism and should not be checked for a "Use when …"
+    phrase in the description.
+    """
+    fm = skill.frontmatter
+    if fm.get("disable-model-invocation") is True:
+        return False
+    paths = fm.get("paths")
+    if paths is not None and paths != "":
+        return False
+    return True
```

Ruff (0.15.12)

[warning] Return the condition `not (paths is not None and paths != "")` directly

Replace with `return not (paths is not None and paths != "")`

(SIM103)